beacon-chain/core/epoch/epoch_operations.go

```diff
@@ -225,21 +225,27 @@ func TotalBalance(
 //    Let inclusion_slot(state, index) =
 //    a.slot_included for the attestation a where index is in
 //    get_attestation_participants(state, a.data, a.participation_bitfield)
+//    If multiple attestations are applicable, the attestation with
+//    lowest `slot_included` is considered.
 func InclusionSlot(state *pb.BeaconState, validatorIndex uint32) (uint64, error) {
-
+	lowestSlotIncluded := uint64(1 << 63)
```

Comment:
Is this supposed to be math.MaxUint64? https://golang.org/pkg/math/#pkg-constants

It would be 1<<64 - 1, or just use math.MaxUint64 :)